Objects/typeobject.c

```diff
@@ -5409,6 +5409,13 @@ PyType_Ready(PyTypeObject *type)
         _PyObject_ASSERT((PyObject *)type, type->tp_vectorcall_offset > 0);
         _PyObject_ASSERT((PyObject *)type, type->tp_call != NULL);
     }
+    /* Consistency check for Py_TPFLAGS_HAVE_SEND - flag requires
+     * type->tp_as_async->am_send to be present.
+     */
+    if (type->tp_flags & Py_TPFLAGS_HAVE_SEND) {
```

Comment:
I'd be more specific with the flag name.

```suggestion
    if (type->tp_flags & Py_TPFLAGS_HAVE_AM_SEND) {
```